packages/admin/admin/src/index.ts

```diff
@@ -82,6 +82,7 @@ export { Field, FieldProps } from "./form/Field";
 export { FieldContainer, FieldContainerClassKey, FieldContainerComponent, FieldContainerProps } from "./form/FieldContainer";
 export { AsyncSelectField, AsyncSelectFieldProps } from "./form/fields/AsyncSelectField";
 export { CheckboxField, CheckboxFieldProps } from "./form/fields/CheckboxField";
+export { NumberField, NumberFieldProps } from "./form/fields/NumberField";
```

Comment:
@jamesricky should we export FinalFormNumberInput as well?

Reply:
It seems to me that @jamesricky agrees: https://github.com/vivid-planet/comet/pull/1938/files/45bec6b8ae45d037ec43274bd9f6884468a2354e#r1615523453

Reply:
Actually I'm not so sure there is any use-case for this, since you can override anything in `NumberField` with the props anyway 😅

Reply:
But we do it for all other components. IMO we should do it here as well.